llvm/test/Transforms/InstCombine/win-math.ll

```diff
@@ -333,3 +333,15 @@ define float @float_powsqrt(float %x) nounwind readnone {
     %1 = call ninf float @powf(float %x, float 0.5)
     ret float %1
 }
+
+define fp128 @fdim_fp128(fp128 %x, fp128 %y) nounwind readnone{
+; CHECK-LABEL: define fp128 @fdim_fp128(
+; MINGW32-NOT: fp128 @fdiml
+;
+  %1 = call fp128 @fdiml(fp128 %x, fp128 %y)
```

Comment:
I don't understand this test; what positive transform would happen? This is an ordinary, unoptimizable call?

Reply:
yh, unoptimizable call
tbh i thought windows will not generate `fdiml` and that would be enough
i will try to other 2 ways, but i think i will need some help, i am new to the codebase

Reply:
In your constant folding patch, this wouldn't constant fold on windows. so you could test it there I suppose

Reply:
alright, I will just remove the 2nd commit from this PR